lib/fabrik/DirectorManager.js

```diff
@@ -423,7 +448,8 @@ class DirectorManager extends BaseManager {
       previous_manifest: opts.previous_manifest,
       multi_az_enabled: config.multi_az_enabled,
       stemcell: this.stemcell,
-      actions: preDeployResponse
+      actions: preDeployResponse,
+      preUpdateActionResponse: preUpdateResponse
```

Comment:
Dont think these params `preDeployResponse` & `preUpdateResponse` are intuitive enough and might affect the code readability. Can we try to think of naming / grouping them in a better way if possible? if we cant then fine :-)

Reply:
Changed `preUpdateResponse` to `preUpdateAgentResponse`